push/ups_integration_with_openshift.md

```diff
@@ -0,0 +1,100 @@
+# Unified Push Server Integration with OpenShift 
+
+## Introduction
+
+This proposal outlines some developer experience issues with provisioning, setting up and configuring the Unified Push Server (UPS) on OpenShift, and integrating it with Apps using the Aerogear SDK.
+
+## Problem Description
+
+The Unified Push Server is made up of a Java application & a database (MySQL).
+It also has a dependecy on an auth provider (currently Keycloak) for users of the UPS Administration UI or REST API.
+There are docker images available for UPS.
+There is also a proof of concept UPS Ansible Playbook Bundle (APB) available.
+However, the APB just provisions UPS and MySQL currently.
+It expects an existing Keycloak instance to be running somewhere with the appropriate Realm, Client & Roles already setup.
+
+The second problem is the lack of a consistent login experience when compared to other services (e.g. Jenkins, Prometheus, Grafana).
```

Comment:
these services are all "just" protected using Openshift - not Keycloak, right ?

Reply:
Correct